### PR TITLE
Replace /gtfs-date with /gtfs-feed-info for more descriptive date info

### PR DESCRIPTION
**Pull request by alanna-zhou**
## Overview
Last semester, I created a `/gtfs-date` endpoint when we did automatic fetching of GTFS data to know when exactly we last fetched the data. However, we are no longer doing automatic fetching, but I discovered that there is a `feed_info.txt` in our GTFS data that is actually super helpful. It tells us the feed start date, feed end date, and feed version. So, I decided to scrap the old `/gtfs-date` with a new `/gtfs-feed-info`.

And now that we know when the GTFS data is no longer valid for (feed end date), it'd be great to have `integration` ping this endpoint to:
* slack the transit-integration-results channel that the GTFS data is about to expire
* build a new image of the microservice and ghopper and deploy them to their servers

I go more in detail about why this is necessary in the git issue linked below.

I also ran the pre-commit formatter!

## Changes Made
I used a csv reader to parse the `feed_info.txt` file in our `tcat-ny-us` folder that holds our GTFS data. I got the dates that correspond only to the keys "feed_start_date", "feed_end_date", and "feed_version." Then, I updated our return variable to a list that actually contains these values, as well as the endpoint name to make more sense.

## Test Coverage
Tested this locally on Postman. Because we will never upload an image of the micro-service without the GTFS data already in it, and because this endpoint doesn't have anything to do with the client, there's no urgent need to deploy this onto the servers to see if it works. I will still however, let it sit a couple of days on the dev server before putting it on prod once I get enough feedback on this PR.

## Next Steps & Related Issues
This is intended to get [the issue in `integration`](https://github.com/cuappdev/integration/issues/14) moving.

## Screenshots

<details>

  <summary>Postman result from running locally</summary>
  
</details>
Screen Shot 2019-10-16 at 5 43 56 PM

### CODE REVIEW

File: src/gtfs.py [hunk 1]
```diff
@@ -25,11 +20,19 @@ def extract_gtfs():
             route = {}
             for index, column in enumerate(column_names):
                 route[column] = row[index]
-            gtfs_data.append(route)
+                gtfs_data.append(route)
```
Comment (kq-li) [inline src/gtfs.py]: seems wrong

File: src/gtfs.py [hunk 2]
```diff
@@ -25,11 +20,19 @@ def extract_gtfs():
...
-def get_gtfs_date_updated():
-    return date_updated
+def get_gtfs_feed_info():
+    global gtfs_feed_info
+    with open(f"{TCAT_NY_US}/feed_info.txt") as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=",")
+        column_names = next(csv_reader)
+        for row in csv_reader:
+            for index, column in enumerate(column_names):
+                if column in ["feed_start_date", "feed_end_date", "feed_version"]:
```
Comment (young-k) [inline src/gtfs.py]: could be `{}` instead of `[]`

File: src/gtfs.py (hunk 1 above)
Comment (alanna-zhou) [reply src/gtfs.py]: oops idk how that got there but nice catch thanks

File: src/gtfs.py (hunk 2 above)
Comment (alanna-zhou) [reply src/gtfs.py]: as in `{"feed_start_date", "feed_end_date", "feed_version"}`?

Comment (alanna-zhou) [reply src/gtfs.py]: why would it be better to have `{}` instead of `[]`?

Comment (meganle) [reply src/gtfs.py]: `{}` indicates set instead of list

**Pull request merged**